ospfd/ospf_te.c

```diff
@@ -2246,6 +2246,10 @@ static int ospf_te_parse_te(struct ls_ted *ted, struct ospf_lsa *lsa)
 	}
 
 	/* Get corresponding Edge from Link State Data Base */
+	if (IPV4_NET0(attr.standard.local.s_addr)) {
+		ote_debug("  |- Found no valid TE Link local address. Abort!");
+		return -1;
+	}
 	edge = get_edge(ted, attr.adv, attr.standard.local);
```

Comment:
can we validate the "edge" instead of the above added check?

Reply:
No. In fact, the edge structure is correctly created. But, the unique identifier of the edge in the TED is based on the local address. Thus, if the local address is IPV4_NET0 (i.e. 0.0.0.0) we cannot correctly reference the edge in the TED. However, local ID is also a valid identifier. So, I need to check that both local address and local ID have not been found during TE parsing. I'll update the PR accordingly.